test/extended/topology_manager/resourcealign.go

```diff
@@ -260,6 +260,11 @@ var _ = g.Describe("[Serial][sig-node][Feature:TopologyManager] Configured clust
 			})
 
 			// rhbz#1813567
+			// a well-behaving TM must *NOT* admit either of the pods.
```

Comment:
Should this be, a well-behaving TM must admin only 1 of the pods?

Reply:
Yep, bad wording. Will rephrase and fix.